tests/test_config.py

```
@@ -57,6 +59,55 @@ def setUp(self):
+                )
+            args = MagicMock(config=config_path)
+            logger = MagicMock()
+
+            with patch.object(mmrelay.config, "logger", logger):
+                loaded = load_config_silently(args)
+
+            self.assertEqual(
+                loaded,
+                {
+                    "matrix": {
+                        "credentials_path": "/tmp/credentials.json",
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/credentials.json"

(S108)